packages/tests/tests/peer_exchange.node.spec.ts

```diff
@@ -1,60 +1,69 @@
 import { waitForRemotePeer } from "@waku/core/lib/wait_for_remote_peer";
-import { createLightNode } from "@waku/create";
-import type { WakuLight } from "@waku/interfaces";
+import { createFullNode } from "@waku/create";
+import type { PeerExchangeResponse, WakuFull } from "@waku/interfaces";
 import { Protocols } from "@waku/interfaces";
 
-import { makeLogFileName, Nwaku } from "../src";
+import { NOISE_KEY_1, Nwaku } from "../src";
 import { delay } from "../src/delay";
 
 describe("Peer Exchange: Node", () => {
-  let waku: WakuLight;
+  let waku: WakuFull;
   let nwaku1: Nwaku;
   let nwaku2: Nwaku;
   let nwaku3: Nwaku;
-
   afterEach(async function () {
     !!nwaku1 && nwaku1.stop();
     !!nwaku2 && nwaku2.stop();
     !!nwaku3 && nwaku3.stop();
+
     !!waku && waku.stop().catch((e) => console.log("Waku failed to stop", e));
   });
 
-  it("Queries successfully", async function () {
-    this.timeout(15_000);
+  it.only("Queries successfully", async function () {
+    this.timeout(150_000);
 
     console.log("starting");
-    nwaku1 = new Nwaku(makeLogFileName(this));
-    nwaku2 = new Nwaku(makeLogFileName(this));
-    nwaku3 = new Nwaku(makeLogFileName(this));
+    nwaku1 = new Nwaku("node1");
+    nwaku2 = new Nwaku("node2");
+    nwaku3 = new Nwaku("node3");
 
-    await nwaku2.start({ discv5Discovery: true, peerExchange: true });
-    console.log("nwaku2 started");
+    await nwaku1.start({
+      discv5Discovery: true,
+      peerExchange: true,
+    });
+    console.log("nwaku1 started");
 
-    const nwaku2Enr = (await nwaku2.info()).enrUri;
+    await delay(10000);
 
-    if (!nwaku2Enr) {
-      throw new Error("No ENR");
-    }
+    const enr = (await nwaku1.info()).enrUri;
+    console.log({ enr });
 
-    await nwaku1.start({
+    await nwaku2.start({
       discv5Discovery: true,
-      discv5BootstrapNode: nwaku2Enr,
     });
-    console.log("nwaku1 started");
+    console.log("nwaku2 started");
+
+    await delay(10000);
 
     await nwaku3.start({
       discv5Discovery: true,
```

Comment:
not using discv5 bootstrap anymore?

One question: What logs would be displayed in nwaku logs to show that it found a peer via discv5 and hence, it would return it in peer exchange?

If there is such log, then we could wait on it instead of a random delay defined in seconds.

Reply:
cc @kaiserd could you help answer this? I haven't been able to discover a discv5 peer